src/main/java/frc/robot/drivetrains/simpleTankDrivetrain/OI.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.drivetrains.simpleTankDrivetrain;
+
+import edu.wpi.first.wpilibj.Joystick;
+
+/**
+ * Handles the communication between the user and the robot.
+ */
+public class OI /*GEVALD*/ {
+
+    private final Joystick leftJoystick = new Joystick(0);
+    private final Joystick rightJoystick = new Joystick(1);
+
+    public OI() {
+    }
+
+    public double getLeftX() {
```

Comment:
add `getRightX()` and `getLeftY()`. Even if they're not used, a team that copy-pastes the code might want to use the joysticks differently